src/WindowManager.vala

```diff
@@ -76,6 +76,11 @@ namespace Gala {
         Clutter.Actor? last_hotcorner;
         public ScreenSaverManager? screensaver { get; private set; }
 
+        /**
+         * Allow to zoom in/out the entire desktop.
+         */
+        Zoom? zoom = null;
```

Comment:
Do we need to declare zoom here? It looks like it isn't ever called outside of `show_stage`